crates/egui/src/containers/panel.rs

```diff
@@ -986,12 +986,12 @@ impl TopBottomPanel {
             let collapsed_height = PanelState::load(ctx, collapsed_panel.id)
                 .map(|state| state.rect.height())
                 .or(collapsed_panel.default_height)
-                .unwrap_or_else(|| ctx.style().spacing.interact_size.y);
+                .unwrap_or_else(|| ctx.active_style().spacing.interact_size.y);
```

Comment:
I think we should go back to calling this just `style` - this produces a lot of diff for little gain.